src/components/structures/BottomLeftMenu.js

```diff
@@ -131,19 +132,19 @@ module.exports = React.createClass({
                     { homeButton }
                     <AccessibleButton className="mx_BottomLeftMenu_people" onClick={ this.onPeopleClick } onMouseEnter={ this.onPeopleMouseEnter } onMouseLeave={ this.onPeopleMouseLeave } >
                         <TintableSvg src="img/icons-people.svg" width="25" height="25" />
-                        { this.getLabel("Start chat", this.state.peopleHover) }
+                        { this.getLabel(counterpart.translate('Start chat'), this.state.peopleHover) }
```

Comment:
So question to all reading: which name for importing should be used? Something like ``translate`` or even shorter something like ``t``?